OracleFMWInfrastructure/README.md

```diff
@@ -0,0 +1,135 @@
+
+        $ sh buildDockerImage.sh
+        Usage: buildDockerImage.sh -v [version] [-s]
+        Builds a Docker Image for Oracle FMW Infrastructure.
+
+        Parameters:
+           -v: version to build. Required.
+           Choose : 12.2.1.2
+           -c: enables Docker image layer cache during build
+           -s: skips the MD5 check of packages
+
+        LICENSE CDDL 1.0 + GPL 2.0
+
+        Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
+
+**IMPORTANT:** the resulting images will have a domain with an Admin Server and one Managed Server by default. You must extend the image with your own Dockerfile, and create your domain using WLST.
+
+
+### Sample FMW Infrastructure Domain 
+The image **oracle/fmw-infrastructure:12.2.1.2** will configure a **base_domain** with the following settings:
+
+ * Admin Username: `weblogic`
+ * Admin Password: `Auto generated` 
+ * Oracle Linux Username: `oracle`
+ * Oracle Linux Password: `welcome1`
+ * Domain Name: `InfraDomain`
+ * Admin Server on port: `7001`
+ * Production Mode: `developer`
+  
+**IMPORTANT:** If you intend to run these images in production you must change the Production Mode to production.
+ 
+
+###Admin Password and Database Schema Password
+
+On the first startup of the container a random password will be generated for the Administration of the domain. You can find this password in the output line:
+
+`Oracle WebLogic Server auto generated Admin password:`
+
+An Oracle Database Schema password will be genrated randomly. You can find this password in the output line:
+
+`Database Schema password Auto Generated :`
+
+If you need to find the passwords at a later time, grep for "password" in the Docker logs generated during the startup of the container.  To look at the Docker Container logs run:
+
+        $ docker logs --details <Container-id>
+
+### Write your own Oracle Fusion Middleware Infrastructure domain with WLST
+The best way to create your own, or extend domains is by using [WebLogic Scripting Tool](https://docs.oracle.com/middleware/1221/cross/wlsttasks.htm). You can find an example of a WLST script to create domains at [createInfraDomain.py](dockerfiles/12.2.1.2/container-scripts/createInfraDomain.py). You may want to tune this script with your own setup to create DataSources and Connection pools, Security Realms, deploy artifacts, and so on. You can also extend images and override an existing domain, or create a new one with WLST.
+
+## Building and Running the Oracle FMW Infrastructure Docker Image
+To try a sample of a FMW Infrastructure Domain image configured, you will need two images, the FMW Infrastructure Domain image and an Oracle Database image. The Oracle Database image can be pulled from the Oracle Container Registry or you can build your own using the Dockerfiles and scripts in GitHub. 
```

Comment:
The Oracle Database image can be pulled from the [Docker Store](https://store.docker.com/images/oracle-database-enterprise-edition) as well as the [Oracle Container Registry](https://container-registry.oracle.com/pls/apex/f?p=113:101). You should also add hyperlinks (as per this comment).